tests/suite/test_prometheus_exporter.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import requests
+
+from kubernetes.client import V1ContainerPort
+
+from suite.resources_utils import wait_until_all_pods_are_ready, ensure_connection
+
+
+@pytest.fixture(scope="class")
+def enable_exporter_port(cli_arguments, kube_apis,
+                         ingress_controller_prerequisites, ingress_controller) -> None:
+    """
+    Set annotations and containerPort for Prometheus Exporter.
+
+    :param cli_arguments: context
+    :param kube_apis: client apis
+    :param ingress_controller_prerequisites
+    :param ingress_controller: IC name
+    :return:
+    """
+    namespace = ingress_controller_prerequisites.namespace
+    annotations = {"prometheus.io/scrape": "true", "prometheus.io/port": "9113"}
```

Comment:
those annotations are for Prometheus metrics collector running in a cluster. Our IC doesn't care about it. Considering that, do we need to add them?

Reply:
No we don't, I'll remove this.